src/control/server/ctl_ranks_rpc_test.go

```diff
@@ -656,7 +656,7 @@ func TestServer_CtlSvc_ResetFormatRanks(t *testing.T) {
 				tc.engineCount = maxEngines
 			}
 
-			ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
It looks like in general the engine changes are more deterministic below. Is it possible to remove the timeout entirely? Could add a conditional just for the test case that is testing the timeout.

Reply:
have addressed in parent PR-8936